lib/src/analysis/example_analyzer.rs

```diff
@@ -0,0 +1,45 @@
+use std::borrow::Cow;
+
+use telcom_parser::lte_rrc::{PCCH_MessageType, PCCH_MessageType_c1, PagingUE_Identity};
+
+use super::analyzer::{Analyzer, Event, EventType, Severity};
+use super::information_element::{InformationElement, LteInformationElement};
+
+pub struct ExampleAnalyzer{
+    pub count: i32,
+}
```

Comment:
imo this kinda thing should be defined in a test file. also this is pedantic i guess but i'd describe it less as an ExampleAnalyzer and more like a DebugAnalyzer

Reply:
Good call, I renamed it test analyzer and put it behind a gate where it will only run for debug builds.

Reply:
ooh perfect